secret-handshake.json

```diff
@@ -0,0 +1,142 @@
+{
+    "#": [
+        "The exercise only lays out the \"commands\" method,", 
+        "but some language tracks have tested reversing the translation and",
+        "composing the two functions, as well.",
+        "Consider the sets not under \"commands\" to be optional."
+    ],
+    "commands": {
+        "description": ["Create a handshake for a number"],
+        "cases": [
+            {
+                "description": "shake for 1",
+                "input": 1,
+                "expected": ["wink"]
+            },
+            {
+                "description": "shake for 10",
+                "input": 2,
+                "expected": ["double blink"]
+            },
+            {
+                "description": "shake for 100",
+                "input": 4,
+                "expected": ["close your eyes"]
+            },
+            {
+                "description": "shake for 1000",
+                "input": 8,
+                "expected": ["jump"]
+            },
+            {
+                "description": "shake for 11",
+                "input": 3,
+                "expected": ["wink", "double blink"]
+            },
+            {
+                "description": "shake for 11 reversed",
+                "input": 19,
+                "expected": ["double blink", "wink"]
+            },
+            {
+                "description": "shake for all",
+                "input": 16,
```

Comment:
although I gotta say, it's going to be good to have a test for 16 as well (the result should be empty list for 16, right?)